abap-code-review/README.md

```diff
@@ -0,0 +1,27 @@
+# ABAP Code Review Guide
+ABAP Code Review Guide is a collection of ideas, tools, and approaches for implementing ABAP code reviews in various landscapes.
```

Comment:
Should some sentences be added that the setup there are currently some investments done and the setup will probably change in the coming year?

Reply:
??? so SAP is moving in a different direction, which is not the one described??

Reply:
This guide is independent of any ongoing discussions on technical foundations how to add code reviews, e.g., to classical CTS flows. --> no need to add confusion here